hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/WriteStatus.java

```diff
@@ -257,9 +274,15 @@ public boolean isTrackingSuccessfulWrites() {
     return trackSuccessRecords;
   }
 
+  public boolean isUpdatesMetadataTable() {
+    return updatesMetadataTable;
+  }
+
   @Override
   public String toString() {
-    return "WriteStatus {" + "fileId=" + fileId
+    return "WriteStatus {"
+        + "isUpdatesMetadataTable=" + updatesMetadataTable
```

Comment:
isMetadataTable

Reply:
I changed the name based on this comment. https://github.com/apache/hudi/pull/13305#discussion_r2122506445 
Should I change it to `isForMetadataTable`?